src/Controller/DefaultController.php

```diff
@@ -40,6 +41,15 @@ public function nasIndex(FundingOrgFilter $fundingOrgFilter)
 
         $fundingCycles = $this->get('doctrine')->getRepository(FundingCycle::class)->findBy($filter, array('name' => 'ASC'));
 
+        $leadershipPersons = $this->get('doctrine')->getRepository(PersonResearchGroup::class)->getLeadershipPeople();
```

Comment:
I technically doesn't matter in what custom repository you write a certain function, in the case: `getLeadershipPeople`.
It just may be unexpected to find this in `PersonResearchGroup`, maybe this should be a function of `ResearchGroup` or `People` (probably research group). It doesn't matter that much, but it will be easier to find, and maybe more logical?

Reply:
Yeah, we can refactor as an improvement